packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js

```diff
@@ -443,7 +443,7 @@ export class SearchFormImpl extends React.PureComponent {
             type="number"
             component={AdaptedInput}
             placeholder="Limit Results"
-            props={{ disabled, min: 1, max: 1500 }}
```

Comment:
does the default-config below provide defaults per value, or only as a whole configuration?

Reply:
It takes default per value.
If we configure maxLimit as 500, It will take 500.  If the value "maxLimit" isn't configure, it will take the default value, in this case, 1500.
@everett980 could you tell me if I am wrong?
Thank you!

Reply:
@gabrieldyck you are correct. `default-config` is consumed per value.